src/test/java/com/marginallyclever/makelangelo/turtle/TurtleTest.java

```diff
@@ -269,4 +270,16 @@ public void notEqualsTwoTurtles() {
         assertNotEquals(turtle, turtle2);
     }
 
+
+    @Test
+    public void testInterpolate() {
+        Turtle turtle = new Turtle();
+        turtle.penDown();
+        turtle.forward(1000);
+        double d = turtle.getDrawDistance();
+        assertEquals(1000,d);
+        assertTrue(new Point2D(0,0).distance(turtle.interpolate(0))<1e-6);
```

Comment:
`1e-6` what is this number ? Please add it as a constant with an explicit name to understand what it means

Reply:
when comparing distances there is often a very very small rounding error in doubles.  so this epsilon value makes sure there is no mistake.  I made it into a constant.